src/state/queries/profile.ts

```diff
@@ -32,6 +32,7 @@ export function useProfileQuery({did}: {did: string | undefined}) {
     // if you remove it, the UI infinite-loops
     // -prf
     staleTime: STALE.MINUTES.FIVE,
+    refetchInterval: STALE.MINUTES.FIVE,
```

Comment:
Five is good for viewing other user's profiles, but for your own, we may want to conditionally lower this to like 30s? @pfrazee thoughts?

Reply:
Mmm yeah that might be the move